### DISCUSSION TITLE
请问7.2.4更新了什么？或者更新日志在哪里可以读？

### DISCUSSION THREAD

**Discussion by Heqilian** [tuna/thuthesis#751] [Q&A] [answered]
我用7.2.3写的论文，需不需要换成新版本？

**Comment by zepinglee** [comment, marked as answer]
https://github.com/tuna/thuthesis/releases/tag/v7.2.4

https://github.com/tuna/thuthesis/blob/master/CHANGELOG.md

**Comment by Harry-Chen** [comment]
7.2.4 没有引入大的改动（写作指南也没有显著的改动）。即便如此，我们总是推荐用户更新到最新版
